src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  setMovie: React.Dispatch<React.SetStateAction<Movie[]>>;
+  movies: Movie[];
+};
+
+export const FindMovie: React.FC<Props>
+= ({ setMovie, movies }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
+  const [errorTitle, setErrorTitle] = useState(false);
+
+  const defaultPoster
+  = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+  };
+
+  const handleClickFindMovie
+  = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    if (query) {
+      getMovie(query)
+        .then(res => {
+          if ('Error' in res) {
+            setFoundMovie(null);
+            setErrorTitle(true);
+          } else {
+            const newMovie = {
+              title: res.Title,
+              description: res.Plot,
+              imgUrl: res.Poster || defaultPoster,
+              imdbId: res.imdbID,
+              imdbUrl: `https://www.imdb.com/title/${res.imdbID}`,
+            };
+
+            setFoundMovie(newMovie);
+            setErrorTitle(false);
+          }
+        });
+    }
+
+    setIsLoading(false);
```

Comment:
move it to `.finally` because now the loading circle doesn't show
![image](https://user-images.githubusercontent.com/86923089/208987112-f45b84e0-4030-4608-a668-358a8cacb07b.png)